src/ui/server/helpers/date/format-date/index.test.ts

```diff
@@ -0,0 +1,14 @@
+import { format } from 'date-fns';
+import formatDate from '.';
+
+describe('server/helpers/date/format-date', () => {
+  const mockTimestamp = new Date();
+
+  it('should return a formatted date', () => {
```

Comment:
A test when `undefined` or `null` argument is passed?

Reply:
@abhi-markan We don't need to 😃 the `timestamp` parameter is required - if there is an attempt to call this function without it, the code won't compile because of typescript. 

We could add a test for this, but doesn't seem to be value in doing this, would also have to add `ts-ignore` because it would flag that the parameter is required